src/light_client.rs

```diff
@@ -268,8 +268,23 @@ pub async fn run(
 							.collect::<Vec<_>>();
 						rpc_fetched.extend(partition_fetched_filtered.clone());
 					}
+
+					// TODO: Skip erasure coded rows from rpc_fetched
+					// e.g. let rpc_fetched_data = rpc_fetched.iter().filter(...)
+					let rpc_fetched_data_rows = data::rows(&dimensions, &rpc_fetched);
```

Comment:
Should the LC be responsible for row insertions? Or does it make more sense for the app client to do it, because the rows are being used solely by the app client?

Reply:
Its actually "fat" client, it will push rows from partition if partition is configured. Maybe it could be additional configuration, but maybe it makes sense to push everything upfront.